### DISCUSSION TITLE
Trying to rg a file that doesn't exist, rg gives no error

### DISCUSSION THREAD

**Discussion by smprather** [BurntSushi/ripgrep#1998] [Q&A] [answered]
Hi RG--

I'm wondering if this is intentional? When giving a target file|dir that doesn't exist, rg gives no error. Thanks!

```
% ls foo
ls: cannot access foo: No such file or directory
% grep bar foo
grep: foo: No such file or directory
% ~/bin/rg bar foo
%
```

**Comment by BurntSushi** [comment, marked as answer]
I cannot reproduce it:

```
$ rg --version
ripgrep 13.0.0 (rev abf115228e)
-SIMD -AVX (compiled)
+SIMD +AVX (runtime)

$ rg bar foo
foo: No such file or directory (os error 2)
```

Can you please run your command with the `--debug` flag? And also run it with the `--no-config` flag? In particular, there are flags that can be given that will suppress error messages. For example:

```
$ rg --no-messages bar foo
$
```

Since you did not really provide enough information to show your complete command, it's not possible to rule out the fact the you may have a config file or an alias somewhere that is setting a flag. You also haven't provided the version of ripgrep that you're using.

**Comment by smprather** [comment]
Sorry to waste your time. User error. I forgot that I was using a wrapper to run rg. Broken wrapper.

% ~/bin/rg.exe bar foo
foo: No such file or directory (os error 2)

Thanks BS! [And thanks for rg! I use it, literally, 50 times a day!]
